boards/arm/mimxrt1064_evk/mimxrt1064_evk.dts

```diff
@@ -141,9 +146,24 @@ arduino_i2c: &lpi2c1 {};
 		spi-max-frequency = <133000000>;
 		status = "okay";
 		jedec-id = [9d 70 17];
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			storage_partition: partition@0 {
+				label = "storage";
+				reg = <0x00000000 DT_SIZE_M(8)>;
+			};
+		};
 	};
 };
 
+&flexspi2 {
+	xip;
```

Comment:
It seems this attribute should be linked to CODE_LOCATION.

Reply:
I'd like to get rid of `CODE_LOCATION` and start using a dts chosen node (`zephyr,flash`) instead. I made an attempt to do that in #30065 but it needs work.